src/util.h

```diff
@@ -13,6 +13,11 @@ extern "C" {
 #include <string.h>
 #include <syslog.h>
 
+// Fallback version tag
+#ifndef NM_VERSION
+#   define NM_VERSION "dev"
```

Comment:
```suggestion
#define NM_VERSION "dev"
```

Just to keep things consistent.